lib/Config.py

```diff
@@ -311,11 +313,18 @@ def getProxy(cls):
         return cls.readSetting("Proxy", "http", cls.default['http_proxy'])
 
     @classmethod
-    def getFile(cls, getfile):
+    def getFile(cls, getfile, compressed=False):
         if cls.getProxy():
             proxy = req.ProxyHandler({'http': cls.getProxy(), 'https': cls.getProxy()})
             auth = req.HTTPBasicAuthHandler()
             opener = req.build_opener(proxy, auth, req.HTTPHandler)
             req.install_opener(opener)
-        return req.urlopen(getfile)
+        if not compressed:
+            return req.urlopen(getfile)
+        else:
+            response = req.urlopen(getfile + '.gz')
+            if 'gzip' in response.info().get('Content-Type'):
+                buf = BytesIO(response.read())
+                data = gzip.GzipFile(fileobj=buf)
+            return (data, response)
```

Comment:
If for some reason "gzip" is _not_ in `Content-Type` then `data` won't be set and this will raise a `ValueError`. You could either do away with the check or set data to `None` just before the if.